pkg/debugapi/debugapi.go

```diff
@@ -145,3 +149,54 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	h.ServeHTTP(w, r)
 }
+
+type noOpSwap struct {
```

Comment:
So I went through the APIs to see how they are handled. Seems like there are a few errors like ErrNoChequebook which are being handled. I wonder if we should retain the errors here instead of returning empty responses? If API expects to send No Chequebook error if there is no chequebook, then this would sort of break that no?